tests/actions/ReportTest.js

```diff
@@ -0,0 +1,73 @@
+import moment from 'moment';
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../src/ONYXKEYS';
+import * as Pusher from '../../src/libs/Pusher/pusher';
+import PusherConnectionManager from '../../src/libs/PusherConnectionManager';
+import CONFIG from '../../src/CONFIG';
+import {subscribeToReportCommentEvents} from '../../src/libs/actions/Report';
+import waitForPromisesToResolve from '../utils/waitForPromisesToResolve';
+
+describe('actions/Report', () => {
+    it('should subscribe to the private-user-accountID channel and handle reportComment events', () => {
```

Comment:
I think this description can be improved to explain "why" this is being tested. Specifically, I think it's testing that when a report action is added, then the resulting Pusher event results in the new report action being put into Onyx. The description of which channel is being subscribed to or the name of the event being handled is secondary information.

Reply:
Yeah I agree that this could use some more emphasis on the relevant info.